### PR TITLE
PCG_12_55_Home-Page-Syntax-Error

### LINKED ISSUE #55: Bug: Home Page Syntax Error
This issue is to fix the syntax error on home page.

### PR DESCRIPTION
**Pull request by JoyZhang2023**
Resolves #55 
This pull request is to fix the syntax error on home page so it could render properly.
image

### CODE REVIEW

**Review by larsswenson — APPROVED** [review]
I'd noticed this error and was planning to add it to the backlog, so well done and thank you!

File: app/page.tsx [hunk 1]
```diff
@@ -75,6 +61,19 @@ export default function Home() {
...
     </ImageList>
+    <Card sx={{ height: 362.359, backgroundImage: "url('/images/index/background_inner.jpg')", backgroundPosition:'center', backgroundColor:"rgba(0,0,0,0.4)", marginTop: 5 }}>
+      <CardContent sx={{alignContent: "center"}}>
+        <Typography color="white" component="h1" fontSize="36px" sx={{ pb: 2, textAlign: "center", marginTop:2, }}>EXPERIENCE MATTERS.</Typography>
+        <Typography color="white" variant="body1" fontSize="22px" sx={{ pb: 4, textAlign: "center", marginLeft:"10%", marginRight:"10%" }}>
+        We specialize in all aspects of home remodeling and improvements. Bathrooms, kitchens, decks, fences, sheds, finishing basements, and most other home improvements or repairs. Our goal is the provide the best customer experience and satisfaction in the industry. Give us a call for a free estimate.
+        </Typography>
+      </CardContent>
+      <CardActions disableSpacing>
+        <Button variant="contained" color="primary" href="/contact" sx={{ fontSize: "1.2rem", width: "300px", height: "50px", textAlign: "center", marginLeft:"auto", marginRight:"auto" }}>
```
Comment (ken-ni) [inline app/page.tsx:72]:
```suggestion
        <Button variant="contained" color="primary" href="/estimates" sx={{ fontSize: "1.2rem", width: "300px", height: "50px", textAlign: "center", marginLeft:"auto", marginRight:"auto" }}>
```

**Review by ken-ni — CHANGES_REQUESTED** [review]
This looks great! Just requesting a small change, I believe this section is supposed redirect to the estimates page, not the contact page since the contact us at the top redirects the user to the contacts page already.

File: app/page.tsx (hunk 1 above)
Comment (JoyZhang2023) [reply app/page.tsx]: Thanks for catching that. Just update the link and push it. Let me know if there's any problem.

**Review by ken-ni — APPROVED** [review]
LGTM!

**Pull request merged**